SPECS/fipscheck/fipscheck.spec

```diff
@@ -0,0 +1,156 @@
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Summary:	A library for integrity verification of FIPS validated modules
+Name:		fipscheck
+Version:	1.5.0
+Release:	9%{?dist}
+License:	BSD
+# This is a Red Hat maintained package which is specific to
+# our distribution.
+URL:		https://pagure.io/fipscheck
+Source0:	https://releases.pagure.org/%{name}/%{name}-%{version}.tar.bz2
+
+BuildRequires:	gcc
+BuildRequires:	openssl-devel >= 1.0.0
+
+Requires:      %{name}-lib%{?_isa} = %{version}-%{release}
+
+%description
+FIPSCheck is a library for integrity verification of FIPS validated
+modules. The package also provides helper binaries for creation and
+verification of the HMAC-SHA256 checksum files.
+
+%package lib
+Summary:	Library files for %{name}
+
+Requires:	%{_bindir}/fipscheck
+
+%description lib
+This package contains the FIPSCheck library.
+
+%package devel
+Summary:	Development files for %{name}
+
+Requires:	%{name}-lib%{?_isa} = %{version}-%{release}
+
+%description devel
+This package contains development files for %{name}.
+
+%prep
+%setup -q
+
+%build
+%configure --disable-static
+
+make %{?_smp_mflags}
+
+# Add generation of HMAC checksums of the final stripped binaries
+%define __spec_install_post \
+    %{?__debug_package:%{__debug_install_post}} \
+    %{__arch_install_post} \
+    %{__os_install_post} \
+    $RPM_BUILD_ROOT%{_bindir}/fipshmac -d $RPM_BUILD_ROOT%{_libdir}/fipscheck $RPM_BUILD_ROOT%{_bindir}/fipscheck $RPM_BUILD_ROOT%{_libdir}/libfipscheck.so.1.2.1 \
```

Comment:
One thing to watch out for when running the spec linter (unless that was fixed while I was on vacation): the linter may attempt to replace `%{_libdir}` and its 64 bit counterpart with `%{_lib}`, which is not the same thing. We need to stick to the `*dir` ones.

Reply:
Seems it is fixed. linter didn't replace `%{_libdir}` with `%{_lib}`.